zerver/openapi/zulip.yaml

```diff
@@ -1419,10 +1434,22 @@ paths:
                                 [GET /streams](/api/get-streams#response) response
                                 for details on the various properties of a channel.
 
-                                **Changes**: Before Zulip 9.0 (feature level 256), this event
-                                was never sent when the `first_message_id` property of a channel
-                                was updated because the oldest message that had been sent to it
+                                This event is also sent when archiving or unarchiving a
```

Comment:
Previously, we did not mention about creation event being sent on unarchiving a stream and neither there is a "Changes" entry for that change.